grr/gui/webauth.py

```diff
@@ -88,6 +88,29 @@ def SecurityCheck(self, func, request, *args, **kwargs):
     return response
 
 
+class HTTPWebAuthManager(BaseWebAuthManager):
```

Comment:
Do you mind adding a simple test for this manager to https://github.com/google/grr/blob/master/grr/gui/webauth_test.py?